content/en/internal/configuration-database/access/_index.md

```diff
@@ -1,64 +1,65 @@
 ---
 title: "Access"
-description: "Accessing GOCDB"
-weight: 20
+description: "Accessing the configuration database"
+weight: 25
 type: "docs"
 ---
 
-The GOCDB can be accessed at [https://goc.egi.eu](https://goc.egi.eu).
+To access [web interface](https://goc.egi.eu) of the EGI configuration database
+(GOCDB), users can either:
 
-To access the web interface, users can:
+- [Use EGI Check-in](#using-institutional-account-via-egi-check-in) to access
+  the GOCDB with an institutional account, or
+- [Use an X.509 digital certificate](#using-an-x509-digital-certificate)
+  installed in their browser.
 
-- use EGI Check-in to [access the GOCDB with an institutional
-  account](#using-institutional-account-via-egi-check-in).
-- use an [X.509 digital certificate installed in the
-  browser](#using-an-x509-digital-certificate), delivered by
-  one of the recognised EU-Grid-PMA Certification Authorities ;
-
-Users can access the system as soon as they are authenticated however they will
-only be able to update information based on their roles.
+Users can access the system as soon as they are authenticated. However,
+they will only be able to update information based on their roles.
 More information about roles and associated permission is available in the
-[Users and
-roles](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Users_and_roles)
-section.
+[_Users and roles_](https://wiki.egi.eu/wiki/GOCDB/Input_System_User_Documentation#Users_and_roles)
+section of the GOCDB documentation.
 
-All roles applications need to be validated by parent roles or administrators.
-Once this is done, you can access/modify relevant information according to the
-role you have been granted. You can learn more on roles and user accounts by
-reading the #Users and roles section of this documentation.
+Applications requesting a specific role have to be validated by parent roles
+or administrators. Once granted, users can access and/or modify relevant
+information, according to the roles granted to the user.
 
 ## Using institutional account via EGI Check-in
 
 In order to be able to access the GOCDB with their institutional account, users
 need to:
 
 1. Have their Identity Provider (IdP) federated in EGI Check-in (via
-   [eduGAIN](https://edugain.org/) or directly) ;
+   [eduGAIN](https://edugain.org/) or directly)
 1. Have created an [EGI Check-in
-   account](https://wiki.egi.eu/wiki/AAI_usage_guide) ;
+   account](https://wiki.egi.eu/wiki/AAI_usage_guide)
```

Comment:
This should probably point to a page in docs.egi.eu.

Reply:
This comment from the previous review (and a few others) may have been hidden due to the collapse of message by GitHub.

Reply:
I will update this in another revision, the one about check-in. Let's leave it as it is for now, this is a sensible subject.

Reply:
I do not really agree, https://wiki.egi.eu/wiki/AAI_usage_guide is already just a redirection https://docs.egi.eu/users/check-in/, let's be nicer with users and bring them directly to https://docs.egi.eu/users/check-in/ :)

Reply:
Fixed